Marlin/Configuration.h

```diff
@@ -417,8 +417,11 @@ const bool Z_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the logic
                                  // In RAMPS uses servo pin 2. Can be changed in pins file. For other boards pin definition should be made.
                                  // It is assumed that when logic high = filament available
                                  //                    when logic  low = filament ran out
-//const bool FIL_RUNOUT_INVERTING = true;  // Should be uncommented and true or false should assigned
-//#define ENDSTOPPULLUP_FIL_RUNOUT // Uncomment to use internal pullup for filament runout pins if the sensor is defined.
+#ifdef FILAMENT_RUNOUT_SENSOR
+  const bool FIL_RUNOUT_INVERTING = true;  // Should be uncommented and true or false should assigned
+  #define ENDSTOPPULLUP_FIL_RUNOUT // Uncomment to use internal pullup for filament runout pins if the sensor is defined.
```

Comment:
`ENDSTOPPULLUP_FIL_RUNOUT ` was still disabled by default, even with `FILAMENT_RUNOUT_SENSOR` set. Now when you set `FILAMENT_RUNOUT_SENSOR` it automatically enables `ENDSTOPPULLUP_FIL_RUNOUT`. What I want to know is, why is it better to have this enabled by default?